cloudify_agent/resources/script/linux.sh.template

```diff
@@ -1,5 +1,18 @@
 #! /bin/bash -e
 
+
+MYSELF=$(whoami)
+run_as()
+{
+    user=$1
+    if [ $MYSELF == $user ]
+    then
+        ${@:2}
+    else
+        su $user --shell /bin/bash -c "set -e; ${@:2}"
```

Comment:
Why su rather than sudo?

Reply:
because it was su in the code before the changes :P

and that used su rather than sudo because su is installed pretty much always and sudo isn't

Reply:
OK- seems a bit odd because sudo is installed as part of the base image for all the versions we support, and I'm fairly sure we've got other stuff that doesn't work without it I think?

Either way, fair enough.